randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -552,14 +552,17 @@ Extra - room_id: [13]
               # Core-X Requirements
               Missiles
               Combat (Beginner) or 20 DMG Missiles
+              # A way to dodge
+              Combat (Intermediate) or Have Any Jump Upgrade
```

Comment:
I guess we could have something like this, charge beam does do an insane knockback though

Reply:
This is for the core-x. Charge does not do anything to it.